bin/archive_files.py

```diff
@@ -136,6 +138,7 @@
     
     # check if this fire_path is already in the FTP by
     # querying the FIRE API
+    #get just the basename to get the object using S3 API 
     f_in_fire_o = api.fetch_object(firePath=fire_path)
```

Comment:
`fetch_object` may need to be updated to use s3 as well. The current code is not handling when we are trying to upload an existing file in FIRE